Localization/pt-BR.hjson

```diff
@@ -1,3 +1,15 @@
+UI: {
+	Journey: "Jornada"
+	Normal: "Muito Jovem Para Minerar"
+	Expert: "Um Ossinho Não Dói"
+	Master: "Eu Sou A Encarnação de Cthulhu"
```

Comment:
The names of the difficulties seem to be translated well, but I'm questioning whether or not they should be translated at all.. What do you think? I tried translating them to my first language, and either couldn't apply the puns (Expert), couldn't keep obviousness of the references (Master), or couldn't make the punified translation rhyme / be nice to pronounce (Normal.)

Reply:
In Brazilian Portuguese, I feel like they fit pretty well. Have you checked if any versions of Doom (aka what these are referencing) have these translated to your first language? You could then base the translation around that.

Reply:
..After quite some thinking, the best localization I came up with for `Hurt/Bone Me Plenty` in Russian (translated as approximately `Make Me Feel Pain` in Eternal), is pretty much just `Burn Me Alive`, and it's not even a pun on a classic game reference, but more so just a standalone reference to Overhaul, dang it.. Anyway, merging this. Thanks for the PR.